packages/fether-react/src/contexts/AccountContext.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import React from 'react';
+import { Provider, subscribe } from 'react-contextual';
+
+export const consumeAccount = subscribe('account');
+
+export const provideAccount = getAccountAddress => Component => props => {
```

Comment:
I think putting account address in the context is not a good idea.

We already have the URL as our source of truth, we don't need to copy this data into a context, and let components consume the context. Just let the components consume the URL directly.

```javascript
const consumeAccount = withProps( // withProps from recopomse
  ({
    match: {
      params: { accountAddress }
    }
  }) => ({ accountAddress })
)

// ...

@consumeAccount
class MyComp extends React.Component { /* this.props.accountAddress will be defined */}
```